src/Symfony/Component/DependencyInjection/Compiler/RegisterAutoconfigureAttributesPass.php

```diff
@@ -42,7 +44,12 @@ public function accept(Definition $definition): bool
 
     public function processClass(ContainerBuilder $container, \ReflectionClass $class): void
     {
-        foreach ($class->getAttributes(Autoconfigure::class, \ReflectionAttribute::IS_INSTANCEOF) as $attribute) {
+        $attributes = array_merge($class->getAttributes(Autoconfigure::class, \ReflectionAttribute::IS_INSTANCEOF), $class->getAttributes(Lazy::class, \ReflectionAttribute::IS_INSTANCEOF));
+
+        if (1 < \count($attributes)) {
```

Comment:
the implem cannot be the same because it is allowed to have many autoconfigure attributes ;)